engine/jobmaster/dm/dm_jobmaster.go

```diff
@@ -202,7 +204,11 @@ func (jm *JobMaster) OnWorkerStatusUpdated(worker framework.WorkerHandle, newSta
 		return nil
 	}
 	jm.Logger().Info("on worker status updated", zap.String(logutil.ConstFieldWorkerKey, worker.ID()), zap.String("extra bytes", string(newStatus.ExtBytes)))
-	return jm.handleOnlineStatus(worker)
+	if err := jm.handleOnlineStatus(worker); err != nil {
+		return err
+	}
+	jm.taskManager.SetNextCheckTime(time.Now())
```

Comment:
what do we plan to do in next tick?

Reply:
operate-task. when task status changed. e.g. running -> paused